website/docs/guides/data-interactions/data-interactions.mdx

```diff
@@ -12,4 +12,8 @@ After setting up your [ComposeDB Client](../../guides/composedb-client/javascrip
 
 ### Mutations
 
-[**Mutations**](../../guides/data-interactions/mutations.mdx) allow you to create or update data. 
\ No newline at end of file
+[**Mutations**](../../guides/data-interactions/mutations.mdx) allow you to create or update data. 
+
+### Query Filtering and Sorting
```

Comment:
Thanks @PaulLeCam! The main reason why I put this into a separate page is because at the very beginning we have a mutation to create some data for a model so initially I thought it would make sense for people to go through the mutations section first. However, we do cover mutations in the getting-started section of the docs so technically they should be familiar with mutations. I'll move the guide into Queries section, I think it makes the most sense.